integration/oasis/osrmconnector/connector.go

```diff
@@ -0,0 +1,45 @@
+package osrmconnector
+
+import (
+	"github.com/Telenav/osrm-backend/integration/pkg/api/osrm/route"
+)
+
+// RouteResponse contains osrm Route response and error
+type RouteResponse struct {
+	Resp *route.Response
+	Err  error
+}
+
+// OsrmConnector wraps the communication with OSRM server
+type OsrmConnector struct {
```

Comment:
From `Golang`'s convention: it's better to: 
- `OsrmConnector` -> `OSRMConnector`
- `osrmclient` -> `osrmClient`

Reply:
Will do.